common/views/components/CompactCard/CompactCard.js

```diff
@@ -30,6 +31,7 @@ type Props = {|
   DateInfo: ?(Element<typeof DateRange> | Element<typeof EventDateRange>),
   StatusIndicator: ?Element<typeof StatusIndicator>,
   ExtraInfo?: ?Node,
+  xOfY: {| x: number, y: number |},
```

Comment:
The thinking here is that anywhere a component on the site can appear within a list, it may need e.g. its bottom spacing removed if it's the last in the list. The `VerticalSpace` component used within other components can decide whether or not to output the space based on this information.

Reply:
I think this might benefit from being tuned into an `XofY` type.